era5cli/fetch.py

```diff
@@ -157,6 +166,10 @@ def _define_outputfilename(self, var, years):
                   else self.outputprefix)
         yearblock = f"{start}-{end}" if not start == end else f"{start}"
         fname = f"{prefix}_{var}_{yearblock}_{self.period}"
+        if self.area:
+            directions = ['N', 'W', 'S', 'E']
+            coords = [str(c) for c in self._round_area()]
+            fname += '_' + ''.join([d+c for d, c in zip(directions, coords)])
```

Comment:
```suggestion
            fname += '_' + ''.join([c+d for c, d in zip(coords, directions)])
```
I think it's more common to write 40N5E than N40E5 (although google maps will treat them as the same point...)

Reply:
Actually there's another issue here. When you're using N, E, S and W to denote lat/lon, the convention is that N and E are positive, and S and W are negative. So e.g. lat/lon +40/**+5** equals 40N**5E** and +40/**-5** equals 40N**5W**. So it is very unusual to have negative numbers.

Reply:
I removed all references to N,E,S,W to prevent this confusion. Hope this works!